docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -0,0 +1,106 @@
+========================
+OAuth 2.0 authentication
+========================
+
+Trino can be configured to enable OAuth 2.0 authentication over HTTPS for the
+Web UI and the JDBC driver. Trino uses the `Authorization Code
+<https://tools.ietf.org/html/rfc6749#section-1.3.1>`_ flow which exchanges an
+Authorization Code for a token. At a high level, the flow includes the following
+steps:
+
+#. the Trino coordinator redirects a user agent to the Authorization Server
+#. the user authenticates with the Authorization Server, and it approves the Trino's permissions request
```

Comment:
I would leave out `it` in the second clause.  On first read for me, it sounded like we were calling the user `it`

Reply:
I changed it after @mosabua comment: https://github.com/trinodb/trino/pull/6856#discussion_r573258275
@mosabua wdyt?

Reply:
well.. imho it should be there because it is not the user that approves the request.. but the Authroization Server.. thats why "it" should be there

Reply:
I'm fine with either way.